src/App.js

```diff
@@ -1,11 +1,10 @@
-import './App.css';
+import './styles/App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
```

Comment:
- [ ] Your _App_ component _(App.js)_ is not classed based. Kindly change this component to a class-based component.